### PR TITLE
Fix/kiwi image extra version field

### PR DESCRIPTION
**Pull request by evanrolfe**
Fixes https://github.com/openSUSE/open-build-service/issues/4206

### CODE REVIEW

File: src/api/app/models/event/upload.rb [hunk 1]
```diff
@@ -12,33 +12,21 @@ def self.message_bus_routing_key
 
 # == Schema Information
 #
-# Table name: packages
+# Table name: events
```
Comment (mdeniz) [inline src/api/app/models/event/upload.rb]: I think you need to rebase... these changes in the annotation are about the refactoring @Ana06 did to the Events module.

**Comment by mdeniz** [comment]
BTW, why can we edit the preferences in the Software tab? it is intended?

File: src/api/app/models/event/upload.rb (hunk 1 above)
Comment (evanrolfe) [reply src/api/app/models/event/upload.rb]: I have already rebased. It looks like Ana made the changes but didn't commit the changes that annotate made so I'm committing them now.

**Comment by evanrolfe** [comment]
> BTW, why can we edit the preferences in the Software tab? it is intended?

Nope I will fix that in a seperate PR.

File: src/api/app/models/event/upload.rb (hunk 1 above)
Comment (Ana06) [reply src/api/app/models/event/upload.rb]: I had already done this 🤔

Comment (evanrolfe) [reply src/api/app/models/event/upload.rb]: @Ana06 it looks like you might've forgotten to commit some of the event models? For example `app/models/event/upload.rb` still has the wrong table name in the annotate comments:

https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/event/upload.rb

Comment (Ana06) [reply src/api/app/models/event/upload.rb]: yes... 🙈

**Pull request merged**